### PR TITLE
[FR] Ruby 2.2.3, 2.17 and 2.0.0-p647

### PR DESCRIPTION
**Pull request by GRoguelon**
Hi,

Please find the translations of the last news about the recent releases of Ruby.

Thanks.

cc: @chatgris @stomar

### CODE REVIEW

File: fr/news/_posts/2015-08-18-ruby-2-2-3-released.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+layout: news_post
+title: "Ruby 2.2.3 est disponible"
+author: "nagachika"
+translator: "Geoffrey Roguelon"
+date: 2015-08-18 16:00:00 +0000
+lang: fr
+---
+
+La version 2.1.7 de Ruby est disponible.
```
Comment (JuanitoFatas) [inline fr/news/_posts/2015-08-18-ruby-2-2-3-released.md]: 2.2.3?

File: fr/news/_posts/2015-08-18-ruby-2-2-3-released.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Cette version inclut un correctif de sécurité pour une faille sur la vérification
+du nom de domaine de Rubygems.
+Veuillez consulter le lien ci-dessous pour plus de détails.
+
+* [CVE-2015-3900 Request hijacking vulnerability in RubyGems 2.4.6 and earlier](http://blog.rubygems.org/2015/05/14/CVE-2015-3900.html)
+
+De plus, plusieurs correctifs ont été inclus.
+Consultez les [tickets](https://bugs.ruby-lang.org/projects/ruby-21/issues?set_filter=1&amp;status_id=5)
```
Comment (JuanitoFatas) [inline fr/news/_posts/2015-08-18-ruby-2-2-3-released.md]: links are not for 2.2.3.

**Comment by GRoguelon** [comment]
Hi @JuanitoFatas,

Sorry wrong Copy/paste. It's fixed!

Thanks for your feedbacks.

**Comment by JuanitoFatas** [comment]
@GRoguelon Merci pour votre contributions :+1: :clap: :blush:

**Comment by GRoguelon** [comment]
Avec plaisir!

**Pull request merged**